angular.json

```diff
@@ -34,6 +34,19 @@
           "configurations": {
             "ie": {
               "tsConfig": "src/tsconfig-ie.spec.json"
+            },
+            "saucelabs": {
+              "sourceMap": false,
+              "progress": false,
+              "karmaConfig": "src/karma.sauce.conf.js",
+              "browsers": "CHROME,FIREFOX,EDGE16,EDGE17,SAFARI11,SAFARI12"
```

Comment:
nitpicking → I would have keep that one the `src/karma.sauce.conf.js` file.

Reply:
Good idea actually! Will update